AirCasting/MainTabBarView.swift

```diff
@@ -25,6 +25,7 @@ struct MainTabBarView: View {
     let sessionSynchronizer: SessionSynchronizer
     @StateObject var tabSelection: TabBarSelection = TabBarSelection()
     @StateObject var selectedSection = SelectSection()
+    @StateObject var reorderButton = ReorderButtonTapped()
```

Comment:
can we remove the class `ReorderButtonTapped` since it only has a single property and add it here instead?
smth like:
```Swift
@State private var isCurrentlyReordeing: Bool = false
```